test/testSelenium.js

```diff
@@ -31,6 +33,13 @@
   };
   const testDirectory = './test/output/';
 
+  webPush.setGCMAPIKey('AIzaSyAwmdX6KKd4hPfIcGU2SOfj9vuRDW6u-wo');
+  webPush.setVapidDetails(
```

Comment:
This means that we're going to run all tests with vapid, right? Unless you change tests like:
```
      test('send/receive notification with payload with ' + browser.getPrettyName(), function() {
        this.timeout(PUSH_TEST_TIMEOUT);
        return runTest(browser, {
          payload: 'marco'
        });
});
```
to:
```
      test('send/receive notification with payload with ' + browser.getPrettyName(), function() {
        this.timeout(PUSH_TEST_TIMEOUT);
        return runTest(browser, {
          payload: 'marco',
          vapid: null,
        });
});
```